### PR TITLE
WIP: Added multivariate student t distribution

### PR DESCRIPTION
**Pull request by fonnesbeck**
Just need to add tests. Please have a look.

### CODE REVIEW

**Comment by fonnesbeck** [comment]
Tests are a little tricky, since there is no multivariate T in scipy to test against.

**Comment by fonnesbeck** [comment]
I have tests now, which are fine, but in practice this distribution is not doing very well. If anyone has a chance to take a look, please do.

File: pymc3/examples/ATMIP_2gaussians.py [hunk 1]
```diff
@@ -28,10 +28,10 @@
 def two_gaussians(x):
     log_like1 = - 0.5 * n * tt.log(2 * np.pi) \
                 - 0.5 * tt.log(dsigma) \
-                - 0.5 * (x - mu1).T.dot(isigma).dot(x - mu1)
+                - 0.5 * (x - mu1).tt.dot(isigma).dot(x - mu1)
     log_like2 = - 0.5 * n * tt.log(2 * np.pi) \
                 - 0.5 * tt.log(dsigma) \
-                - 0.5 * (x - mu2).T.dot(isigma).dot(x - mu2)
+                - 0.5 * (x - mu2).tt.dot(isigma).dot(x - mu2)
```
Comment (twiecki) [inline pymc3/examples/ATMIP_2gaussians.py]: bad search/replace

File: pymc3/distributions/continuous.py [hunk 2]
```diff
@@ -576,7 +576,7 @@ class StudentT(Continuous):
 
     Describes a normal variable whose precision is gamma distributed.
     If only nu parameter is passed, this specifies a standard (central)
-    Student's T.
+    Student's tt.
```
Comment (twiecki) [inline pymc3/distributions/continuous.py]: bad search/replace

**Comment by springcoil** [comment]
The tests look fine to me and I don't see any bugs (not saying there aren't any though) in the distribution.

**Comment by fonnesbeck** [comment]
I will merge it so that people can start using it. Can squash bugs as they arise.

**Pull request merged**